src/constants/ActionTypes.js

```diff
@@ -349,6 +349,7 @@ export default utils.deepKeyMirror({
 		SET: {
 			ADD: null,
 			SET_BACKGROUND_LAYER: null,
+            SET_LAYERS: null,
```

Comment:
I'm getting a bit lost in MAPS.SET.SET_LAYERS, MAPS.MAP.LAYER.SET, MAPS.LAYERS.LAYER.SET...
Is there a system, or is it just randomly added?

Reply:
MAPS.SET.SET_LAYERS means set layers in particular MapSet
MAPS.MAP.LAYERS.SET.x means set x property in particular layer in particular Map
I have no idea, what MAPS.LAYERS.LAYER.SET should do